sktime/transformations/series/hurst.py

```diff
@@ -0,0 +1,224 @@
+    long-range dependence.
+
+    Parameters
+    ----------
+    lags : Optional[Union[List[int], range]], default=None
+        The lags to use for calculation. If None, uses a range based on min_lag and max_lag.
+    method : str, default='rs'
+        The method to use for Hurst exponent calculation. Either 'rs' (rescaled range)
+        or 'dfa' (detrended fluctuation analysis).
+    min_lag : int, default=2
+        The minimum lag to use if lags is None.
+    max_lag : int, default=100
+        The maximum lag to use if lags is None.
+    fit_trend : str, default='c'
+        The trend component to include in the calculation.
+    confidence_level : float, default=0.95
+        The confidence level for the confidence interval calculation.
+
+    Attributes
+    ----------
+    hurst_estimate_ : float
+        The estimated Hurst exponent.
+    confidence_interval_ : tuple
+        The confidence interval for the Hurst exponent estimate.
+    """
+
+    _tags = {
+        "scitype:transform-input": "Series",
+        "scitype:transform-output": "Series",
+        "scitype:instancewise": True,
+        "scitype:transform-labels": "None",
+        "X_inner_mtype": "pd.Series",
+        "y_inner_mtype": "None",
+        "univariate-only": True,
+        "requires_y": False,
+        "fit_is_empty": False,
+        "capability:inverse_transform": False,
+        "capability:unequal_length": True,
+        "handles-missing-data": False,
+        "authors": ["phoeenniixx"],
+        "maintainers": ["fkiraly"],
+    }
+
+    def __init__(
+            self,
+            lags: Optional[Union[List[int], range]] = None,
+            method: str = 'rs',
+            min_lag: int = 2,
+            max_lag: int = 100,
+            fit_trend: str = 'c',
+            confidence_level: float = 0.95
+    ):
+        self.lags = lags
+        self.method = method
+        self.min_lag = min_lag
+        self.max_lag = max_lag
+        self.fit_trend = fit_trend
+        self.confidence_level = confidence_level
+
+        super().__init__()
+
+    def _fit(self, X: pd.Series, y=None):
+        """Fit transformer to X and y.
+
+        Parameters
+        ----------
+        X : pd.Series
+            The time series data to fit.
+        y : None
+            Ignored.
+
+        Returns
+        -------
+        self : object
+            Returns self.
+        """
+        self.hurst_estimate_ = self._hurst_exponent(X)
+        return self
+
+    def _transform(self, X: pd.Series, y=None):
+        """Transform X and return a transformed version.
+
+        Parameters
+        ----------
+        X : pd.Series
+            The time series data to transform.
+        y : None
+            Ignored.
+
+        Returns
+        -------
+        X_transformed : pd.Series
+            The input series with its Hurst exponent appended as metadata.
+        """
+        X_transformed = X.copy()
```

Comment:
the return in this case should be a single-row `pd.DataFrame` with columns `hurst_exponent` and other columns, potentially.